docs/user_guide/prometheus_metrics/prometheus_scheduler_metrics.md

```diff
@@ -0,0 +1,283 @@
+---
+id: prometheus_scheduler_metrics
+title: Prometheus Scheduler Metrics
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## container_allocation_attempt_total
```

Comment:
Some subheadings are too long, causing them to be collapsed in the navigation bar. Additionally, due to underlining, words are incorrectly split, making the text less intuitive. To address this, we can either shorten the length of individual headings by using hierarchical titles or simply remove the underlining.
<img width="194" alt="image" src="https://github.com/apache/yunikorn-site/assets/48400525/4b13866f-f31f-4e41-a277-1b39d0bf5032">